tests/test_transforms/test_decomposition/test_change_points_based/test_level.py

```diff
@@ -9,6 +9,7 @@
 from etna.transforms import ChangePointsLevelTransform
 from etna.transforms.decomposition.change_points_based.change_points_models import RupturesChangePointsModel
 from etna.transforms.decomposition.change_points_based.per_interval_models import MeanPerIntervalModel
+from tests.test_transforms.utils import assert_transformation_equals_loaded_original
```

Comment:
Do all the other transforms have this test? It seemed to me that there were more